target/linux/mediatek/mt7622/base-files/etc/board.d/02_network

```diff
@@ -29,9 +30,30 @@ mediatek_setup_interfaces()
 	esac
 }
 
+mediatek_setup_macs()
+{
+	local board="$1"
+	local lan_mac=""
+	local wan_mac=""
+	local label_mac=""
+
+	case $board in
+	ruijie,rg-ew3200gx-pro)
+		wan_mac=$(mtd_get_mac_ascii product_info ethaddr)
```

Comment:
See comment in DTS how to make this a bit better (by setting `ethernet0` alias in DTS instead).